src/main/java/homework2/user/Admin.java

```diff
@@ -0,0 +1,11 @@
+package homework2.user;
+
+public class Admin extends User {
+    @Override
+    public void printRole() {
+        System.out.println("Role admin");
+    }
+    public void checkTicket() {
+        System.out.println("Check ticket method for Admin");
+    }
```

Comment:
checkTicker shouln't print smth - just should check the tickek. for example - do you have ticked by Id

Reply:
Good morning/ i will add everything here.Just I learned better do a lot of small commits)). So in checkTicket i should call method checkTicketById as I understand

Reply:
and maybe I shouln't make main class just call all methods in TicketService